### PR TITLE
adding firefox config settings

### PR DESCRIPTION
**Pull request by sepulworld**
Additional info for Firefox and enabling webrtc

### CODE REVIEW

**Comment by mbalamat** [comment]
@sepulworld thanks! I will change your commit message a bit in order for the checks to pass.

File: README.md [hunk 1]
```diff
@@ -69,6 +69,10 @@ If you have access to a TURN server and you want your balenaCam devices to use i
   - Ensure the media flag is enabled (this will give you access to other features)
   - Now set "Anonymize local IPs exposed by WebRTC" to Disabled
   - Refresh your app page
+- Firefox may also hide local IP address from WebRTC, confirm following in 'config:about'
+  - media.peerconnection.enabled: true
+  - if `media.peerconnection.ice.obfuscate_host_addresses` is set to `true`
```
Comment (mbalamat) [inline README.md:74]: @sepulworld is there a way to disable this host address obfuscation entirely by setting it to false?

**Comment by sepulworld** [comment]
Yes, I tested that and it works as well. You either set that to false or whitelist your camera IP. Either one worked for me. Feel free to update PR or I can do it later this evening

File: README.md (hunk 1 above)
Comment (sepulworld) [reply README.md]: Yep, it works if you set obfuscation to false.

**Comment by mbalamat** [comment]
@balena-ci retest

**Pull request merged**